src/base/components/widgets/Login.js

```diff
@@ -1,70 +1,70 @@
-  );
-  // document.getElementById('loginbuttoniframe').contentWindow.postMessage('login', getServiceUrl('login'));
-};
+// export const performLogin = () => {
+//   window.open(
+//     `${getServiceUrl('login')}/auth/twitter?callback=${encodeURIComponent('/buttons/callback')}`,
+//     'Login',
+//     'height=500,width=700',
+//   );
+//   // document.getElementById('loginbuttoniframe').contentWindow.postMessage('login', getServiceUrl('login'));
+// };
 
-const Login = ({ profile, readItLater = [], onLogout, onLogin }) => (
-  <Dropdown id="dropdown-custom-1" pullRight>
-    <Dropdown.Toggle className="btn-simple btn-default btn-lg btn-flat">
-      <i className="material-icons dp_big">account_circle</i>{' '}
-      {profile.temporary ? 'Guest account' : profile.name}
-    </Dropdown.Toggle>
-    <Dropdown.Menu>
-      {
-        !!readItLater.length && (readItLater.map((o, index) =>
-          <MenuItem href={o.url} key={index}>
-            <i className="material-icons dp_small with_text">bookmark</i>{o.name}
-          </MenuItem>
-        ))
-      }
-      {!!readItLater.length && (<MenuItem divider />)}
-      <MenuItem eventKey="1" href="https://core.wrioos.com/create">
-        <i className="material-icons dp_small with_text">create</i>Create new article
-      </MenuItem>
-      {/*
-        <MenuItem eventKey="1" href="https://core.wrioos.com/create_list">
-        <i className="material-icons dp_small with_text">create</i>Create new list
-        </MenuItem>
-      */}
-      <MenuItem eventKey="1" href={profile.url}>
-        <i className="material-icons dp_small with_text">perm_identity</i>Profile
-      </MenuItem>
-      <MenuItem divider />
-      {!profile.temporary ? (
-        <MenuItem eventKey="2" onClick={performLogout}>
-          <i className="material-icons dp_small with_text">exit_to_app</i>Logout
-        </MenuItem>
-      ) : (
-        <MenuItem eventKey="2" onClick={performLogin} disabled>
-          <i className="material-icons dp_small with_text">exit_to_app</i>Login (coming)
-        </MenuItem>
-      )}
-    </Dropdown.Menu>
-  </Dropdown>
-);
+// const Login = ({ profile, readItLater = [], onLogout, onLogin }) => (
+//   <Dropdown id="dropdown-custom-1" pullRight>
+//     <Dropdown.Toggle className="btn-simple btn-default btn-lg btn-flat">
+//       <i className="material-icons dp_big">account_circle</i>{' '}
+//       {profile.temporary ? 'Guest account' : profile.name}
+//     </Dropdown.Toggle>
+//     <Dropdown.Menu>
+//       {
+//         !!readItLater.length && (readItLater.map((o, index) =>
+//           <MenuItem href={o.url} key={index}>
+//             <i className="material-icons dp_small with_text">bookmark</i>{o.name}
+//           </MenuItem>
+//         ))
+//       }
+//       {!!readItLater.length && (<MenuItem divider />)}
+//       <MenuItem eventKey="1" href="https://core.wrioos.com/create">
+//         <i className="material-icons dp_small with_text">create</i>Create new article
```

Comment:
We need it, see prev PR for the comment.

Reply:
ok sure